### PR TITLE
Add more granular options for play speed in media player

### LINKED ISSUE #392: Add more granular options for play speed in media player
[As requested by DannyFeliz08 on twitter](https://twitter.com/DannyFeliz08/status/1406636065331585028):

> Hey \@EnjoyTheVueCast can you add the support for the speed 1.2x, 1.5x is too fast for me 😅

### PR DESCRIPTION
**Pull request by DannyFeliz**
closes #392

I got these numbers from the Spotify podcast interface, except I didn't add the 3x speed. I think it's extremely fast, and I also didn't want to make the list too long.

image

speed

### CODE REVIEW

File: src/components/MediaPlayer.vue [hunk 1]
```diff
@@ -103,12 +108,18 @@ export default {
...
     },
     changePlaybackRate() {
-      if (this.player.playbackRate === 2) {
-        this.player.playbackRate = 0.5;
+      const playbackRateSpeeds = {
+        0.5: 0.8,
+        0.8: 1,
+        1: 1.2,
+        1.2: 1.5,
+        1.5: 2
```
Comment (fimion) [inline src/components/MediaPlayer.vue:116]: let's add a 1.8 speed between 1.5 and 2.0. We probably don't talk slowly enough for it to be useful, but i definitely listen to audio books at 1.75x speed, so might as well have it here too.

**Review by fimion — CHANGES_REQUESTED** [review]
Looks good! One suggestion.

File: src/components/MediaPlayer.vue (hunk 1 above)
Comment (DannyFeliz) [reply src/components/MediaPlayer.vue]: Done

**Review by fimion — APPROVED** [review]
Approved!
Approved

**Comment by fimion** [comment]
Thanks for the help!

**Comment by DannyFeliz** [comment]
No problem, I love the podcast 🤘🏻, maybe later I will summit more PRs.

**Pull request merged**